src/tools/rbd_mirror/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ add_library(rbd_mirror_types STATIC
 set(rbd_mirror_internal
   ClusterWatcher.cc
   ImageDeleter.cc
+  ImageMapper.cc
```

Comment:
I'd vote to skip class from this for this PR -- it's going to require a lot of refinement before it truly stubs out the necessary interactions